packages/backend/src/studio-api-impl.ts

```diff
@@ -73,9 +72,14 @@ export class StudioApiImpl implements StudioAPI {
     const model = await this.getModelById(modelId);
 
     // Do not wait for the pull application, run it separately
-    this.applicationManager.pullApplication(recipe, model).catch((error: unknown) => console.warn(error));
-
-    return Promise.resolve(undefined);
+    void podmanDesktopApi.window
```

Comment:
can we remove the `void` here?

Reply:
No the reason is avoid linter to complain that we have "dangling promises"

Reply:
for me it runs fine without it 🤔 
Do you mean the `yarn lint:check`?

Reply:
I agree here with @lstocchi we shouldn't use that void to fake the compiler

also you're doing a catch so linter should not report any problem

you can also remove the `<void>` after `.withProgress<void>` as default is void

Reply:
the rule is that you should use:

```ts
await promise // valid
promise.....catch(error...) //valid
```

and not
```ts
promise // invalid (and do not add void before to disable the linter as then at runtime you may have unhandled exception
void promise // no
```